[2021-03-01 20:39:55,450] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-02-11T00:00:00+00:00 [queued]>
[2021-03-01 20:39:55,454] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-02-11T00:00:00+00:00 [queued]>
[2021-03-01 20:39:55,454] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-03-01 20:39:55,454] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-03-01 20:39:55,454] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-03-01 20:39:55,465] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): processing_user> on 2020-02-11T00:00:00+00:00
[2021-03-01 20:39:55,467] {standard_task_runner.py:51} INFO - Started process 240373 to run task
[2021-03-01 20:39:55,469] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'processing_user', '2020-02-11T00:00:00+00:00', '--job-id', '261', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpp2jqw_h2']
[2021-03-01 20:39:55,469] {standard_task_runner.py:76} INFO - Job 261: Subtask processing_user
[2021-03-01 20:39:55,490] {logging_mixin.py:103} INFO - Running <TaskInstance: user_processing.processing_user 2020-02-11T00:00:00+00:00 [running]> on host pop-os.localdomain
[2021-03-01 20:39:55,513] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2020-02-11T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-02-11T00:00:00+00:00
[2021-03-01 20:39:55,519] {python.py:118} INFO - Done. Returned value was: None
[2021-03-01 20:39:55,521] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=processing_user, execution_date=20200211T000000, start_date=20210302T013955, end_date=20210302T013955
[2021-03-01 20:39:55,538] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-01 20:39:55,560] {local_task_job.py:118} INFO - Task exited with return code 0
